org.libredirect.frontends_manager.yml

```diff
@@ -0,0 +1,49 @@
+id: org.libredirect.frontends_manager
+runtime: org.gnome.Platform
+runtime-version: '42'
+command: libredirect-frontends-manager
+rename-icon: libredirect-frontends-manager
+rename-desktop-file: libredirect-frontends-manager.desktop
+
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+  - org.freedesktop.Sdk.Extension.rust-nightly
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-data
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: libredirect-frontends-manager
+    sources:
+      - type: git
+        url: https://github.com/libredirect/frontends_manager.git
+        tag: v1.0.0
+        x-checker-data:
+          type: git
+          tag-pattern: ^v((?:\d+.)*\d+)$
+      - node-sources.json
+      - cargo-sources.json
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/node16/bin:/usr/lib/sdk/rust-nightly/bin
+      env:
+        npm_config_nodedir: /usr/lib/sdk/node16
+        CARGO_HOME: /run/build/libredirect-frontends-manager/cargo
+        npm_config_offline: 'true'
+        npm_config_cache: /run/build/libredirect-frontends-manager/flatpak-node/npm-cache
+    build-commands:
+      - npm ci --offline
+      - cargo --offline fetch --manifest-path src-tauri/Cargo.toml
+      - npm run tauri build -- -d -b deb
+      - install -Dm644 -t /app/share/applications/ src-tauri/target/debug/bundle/deb/*/data/usr/share/applications/*
```

Comment:
why `*` and not just `*.desktop`. Because that's only for desktop files.

Reply:
Fixed.